### ISSUE TITLE
fix: Too many new orders; current limit is 50 orders per 10 SECOND on Binance API

### ISSUE THREAD

**Issue by jessy1092** [c9s/bbgo#228]
I set grid strategy on Binance. it show warning and can not set the correct number of girds.

```
[0004]  WARN <APIError> code=-1015, msg=Too many new orders; current limit is 50 orders per 10 SECOND. strategy=grid
```

My example `bbgo.yml` config
```
exchangeStrategies:
  - on: binance
    grid:
      symbol: DOGEUSDT
      upperPrice: 0.60
      lowerPrice: 0.40
      profitSpread: 0.01
      gridNumber: 100
      quantity: 40
      long: true
```

If I set `gridNumber` as 50. It also shows warning.

**Comment by c9s** [comment]
looks like Binance is pretty strict for running grid

**Comment by c9s** [comment]
any solutions?

**Comment by c9s** [comment]
Fixed in 1.19
